src/openzeppelin/account.cairo

```diff
@@ -1,43 +1,52 @@
 use serde::Serde;
 use array::ArrayTrait;
-use array::SpanTrait;
 use starknet::ContractAddress;
-use starknet::contract_address::ContractAddressSerde;
 use openzeppelin::utils::check_gas;
+use openzeppelin::utils::span_to_array;
 
 const ERC165_ACCOUNT_ID: u32 = 0xa66bd575_u32;
 const ERC1271_VALIDATED: u32 = 0x1626ba7e_u32;
 
 const TRANSACTION_VERSION: felt252 = 1;
-const QUERY_VERSION: felt252 = 340282366920938463463374607431768211457; // 2**128 + TRANSACTION_VERSION
+const QUERY_VERSION: felt252 =
+    340282366920938463463374607431768211457; // 2**128 + TRANSACTION_VERSION
 
+#[derive(Serde, Drop)]
 struct Call {
     to: ContractAddress,
     selector: felt252,
     calldata: Array<felt252>
 }
 
+#[abi]
+trait IAccount {
+    fn __validate__(calls: Array<Call>) -> felt252;
+    fn __validate_declare__(class_hash: felt252) -> felt252;
+    fn __execute__(calls: Array<Call>) -> Array<Array<felt252>>;
+    fn set_public_key(new_public_key: felt252);
+    fn get_public_key() -> felt252;
+}
+
 #[account_contract]
 mod Account {
-    use box::BoxTrait;
     use array::SpanTrait;
     use array::ArrayTrait;
+    use box::BoxTrait;
     use option::OptionTrait;
     use zeroable::Zeroable;
     use ecdsa::check_ecdsa_signature;
+    use serde::ArraySerde;
     use starknet::get_tx_info;
     use starknet::get_caller_address;
     use starknet::get_contract_address;
-    use starknet::contract_address::ContractAddressPartialEq;
-    use starknet::contract_address::ContractAddressZeroable;
+    use debug::PrintTrait;
```

Comment:
how is this used?

Reply:
Just for local debugging, I forgot to remove it.